### DISCUSSION TITLE
Transactions in Prisma

### DISCUSSION THREAD

**Discussion by pbelbin** [prisma/prisma#2767] [Q&A] [answered]
Hey folks,

Any idea when the 'Long-running transactions' support will be available?

https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/transactions#long-running-transactions

**Comment by ryands17** [comment]
Hey @pbelbin :wave: 
Currently we do not have any accurate timeline on when this would be available, but could you add a :+1: to [this](https://github.com/prisma/prisma/issues/1844) feature request and explain the use case that you would have as well in the comments. We would love to know what are the common use cases that people need and that would help use prioritize this request as well :)

**Reply by nikolasburk** [reply, marked as answer]
Note that we are currently evaluating a transactions/batching API here: https://github.com/prisma/prisma-client-js/issues/667#issuecomment-643750802 @pbelbin it would also be great to get your feedback on this API and hear whether that solves your use case 🙌

**Comment by ludralph** [comment]
Hi @pbelbin,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
